source/include/ILDGeometryUtil.h

```diff
@@ -0,0 +1,43 @@
+#ifndef ILDGEOMETRYUTIL_H
+#define ILDGEOMETRYUTIL_H 1
+
+#include <DDRec/DetectorData.h>
+
+
+namespace MarlinUtil{
```

Comment:
```suggestion
namespace MarlinUtil::ILD {
```
I would consider putting these into another (nested) namespace still, just to make it very clear to users that this is ILD specific and might not generally apply to all possible detectors (for which there is still `MarlinUtil::getDetData`). Another advantage is that it will be less likely to get name clashes, if some decides to provide the same functionality in a different header which gets included into the same translation unit. @gaede, do you think that is a clear enough distinction between "general" utilities and ILD specific ones?